src/main/resources/application.yaml

```diff
@@ -0,0 +1,8 @@
+# Camunda 7
```

Comment:
⛏️ Maybe add c8 properties to the application.yaml as comments as well. 
💭 I would briefly mention their existence in the README.

Reply:
💭 Maybe also add the local and SaaS config for C8: 

Local: 

> zeebe.client:
  broker:
    gateway-address: localhost:26500
  security:
    plaintext: true
    
Saas: 
> zeebe.client:
  cloud:
    region: bru-2
    clusterId: 
    clientId: 
    clientSecret: